lerobot/common/robot_devices/robots/configs.py

```diff
@@ -630,6 +630,8 @@ class TrossenAIStationaryRobotConfig(ManipulatorRobotConfig):
     # then to gradually add more motors (by uncommenting), until you can teleoperate both arms fully
     max_relative_target: int | None = 5
 
+    force_feedback_gain: float = 0.0
```

Comment:
Make sure to add a comment describing this parameter similar to `max_relative_target`. Maybe include the suggested value of 0.1 as a good place to start if they want force feedback